### ISSUE TITLE
Cherry tree does not work with screen readers 

### ISSUE THREAD

**Issue by Chana-Miryam** [giuspen/cherrytree#1958]
Cherry tree does not work with screen readers 

Please make it available for NVDA users. Its a screen reader. 
Thank you
